modules/collect/src/main/java/com/opengamma/strata/collect/Guavate.java

```diff
@@ -228,7 +228,7 @@ public boolean hasNext() {
    * @param zipper  the function used to combine the pair of objects
    * @return a stream of pairs, one from each stream
    */
-  private static <A, B, R> Stream<R> zip(Stream<A> stream1, Stream<B> stream2, BiFunction<A, B, R> zipper) {
+  static <A, B, R> Stream<R> zip(Stream<A> stream1, Stream<B> stream2, BiFunction<A, B, R> zipper) {
     // this is private for now, to see if it is really needed on the API
```

Comment:
Does it make sense to leave this comment now that it's used elsewhere in the package?

Reply:
I think so. It explains why it isn't public. And I suspect the new method may end up being a reason to never make it public.